ctru-rs/src/lib.rs

```diff
@@ -3,6 +3,7 @@
 #![feature(test)]
 #![feature(custom_test_frameworks)]
 #![test_runner(test_runner::run)]
+#![feature(once_cell)]
```

Comment:
nit: we can use the once_cell crate instead to avoid adding another feature here.